instantsearch.js/autocomplete-results-page/src/app.js

```diff
@@ -57,13 +57,13 @@ search.addWidgets([
   instantsearch.widgets.hits({
     container: '#hits',
     templates: {
-      item: `
+      item: (hit, { html, components }) => html`
         <div>
           <header class="hit-name">
-            {{#helpers.highlight}}{ "attribute": "name" }{{/helpers.highlight}}
+            ${components.Highlight({ hit, attribute: 'name' })}
```

Comment:
I wonder if it's better to do it the intended `htm`/Preact way like : 
```
item: (hit, { html, components: { Highlight } }) => html`
  <header class="hit-name">
    <${Highlight} hit=${hit} attribute="name" />
  </header>
`
```
However we call these templates as functions anyway so I think they remount every time instead of rerendering anyway.

Reply:
we don't document the htm/preact way yet, and the components don't have state, so I think the way it's written in this pr is good :)

Reply:
Yeah, might be better visually. I think in most examples/tests in instantsearch, the `components` are used like this so I kept it for consistency. Open to changing it.